### PR TITLE
Summary table

### PR DESCRIPTION
**Pull request by vikram-narayan**
add an optional `cost` param to `plot_returns`.  if present, total_returns less cost is plotted.

### CODE REVIEW

**Comment by vikram-narayan** [comment]
@jzhang18 mind taking a quick look?

**Comment by jzhang18** [comment]
@vikram-narayan Sure. I will read it asap.

**Comment by jzhang18** [comment]
@vikram-narayan It looks good to me! Thanks!

**Comment by vikram-narayan** [comment]
Gotcha - @gusgordon should I add that to other plots, and the main perf attrib tearsheet? In a different PR @twiecki said
> As such, we don't need to push that logic to perf_attrib but can just slice the output data as we need it. As such, we should hold off on this PR and wait for the tear-sheet to be pushed where we can add this functionality.

**Comment by gusgordon** [comment]
@vikram-narayan Right, so we just run the perf_attrib function as normal, then whatever it outputs we slice on the `live_start_date` to break the data out.

Definitely want it broken out in the summary table (All, In-sample, Out-of-sample), but I'm not sure about the plots. Might be nice to have some sort of marker for the live start date, like a vertical line? I would say whatever you guys think is useful and looks ok.

File: pyfolio/bayesian.py [hunk 1]
```diff
@@ -86,7 +86,7 @@ def model_returns_t_alpha_beta(data, bmark, samples=2000):
                  mu=mu_reg,
                  sd=sigma,
                  observed=y)
-        trace = pm.sample(samples)
+        trace = pm.sample(samples, progressbar=False)
```
Comment (twiecki) [inline pyfolio/bayesian.py]: Seems like that fixed the travis issue. This isn't a proper fix, however. Can you make this a kwarg for the top function that defaults to `True` and only set it to `False` in the unittest?

Comment (twiecki) [inline pyfolio/bayesian.py]: This seemed to have fixed the issue but it's not a proper fix. Can you move this up to be a kwarg of the function, default to `True` and set it to `False` in the unittest?

File: pyfolio/perf_attrib.py [hunk 2]
```diff
@@ -14,10 +14,11 @@
 # limitations under the License.
 from __future__ import division
 
+from collections import OrderedDict
```
Comment (yankees714) [inline pyfolio/perf_attrib.py]: This should be grouped with the standard lib imports

File: pyfolio/perf_attrib.py [hunk 3]
```diff
@@ -164,46 +169,63 @@ def show_perf_attrib_stats(returns, positions, factor_returns,
         pos_in_dollars=pos_in_dollars,
     )
 
-    perf_attrib_stats = create_perf_attrib_stats(perf_attrib_data)
+    perf_attrib_stats, risk_exposure_stats =\
+        create_perf_attrib_stats(perf_attrib_data, risk_exposures)
```
Comment (yankees714) [inline pyfolio/perf_attrib.py]: Thoughts on having `perf_attrib()` and `create_perf_attrib_stats()` returning things in the same order for consistency? I.e. changing this to `risk_exposure_stat, perf_attrib_stats = ...`

**Comment by joshpayne** [comment]
would it be good on PRs that involve new (or changed) charts and tables in pyfolio (and alphalens) to include screenshots of them in the PR like we do for UI work?

**Comment by jbredeche** [comment]
Yes it would :)

**Pull request merged**